idaes/generic_models/properties/core/state_definitions/tests/test_FPhx_electrolyte.py

```diff
@@ -242,60 +242,53 @@ def test_solve_for_true_species(self, frame):
 
 
 # -----------------------------------------------------------------------------
-def dens_mol_h20(*args, **kwargs):
+def dens_mol_h2o(*args, **kwargs):
```

Comment:
Minor comment because this is just a test, but for chemical symbols I usually follow the typical capitalization, so H2O instead of h2o. For water its not too significant, but NaCl is more readable than nacl.